modules/core/src/copy.cpp

```diff
@@ -599,6 +762,16 @@ flipVert( const uchar* src0, size_t sstep, uchar* dst0, size_t dstep, Size size,
         int i = 0;
         if( ((size_t)src0|(size_t)dst0|(size_t)src1|(size_t)dst1) % sizeof(int) == 0 )
         {
+#if CV_SIMD
+            for( ; i <= size.width - (v_int32::nlanes * 4); i += v_int32::nlanes * 4 )
```

Comment:
SIMD code doesn't require alignment check above, so it can be placed before `if`.